src/components/Venues/Modal/VenueModal.spec.tsx

```
@@ -380,32 +408,95 @@ describe('VenueModal', () => {
+
+      // Verify no image preview is shown after upload failure
+      expect(
+        screen.queryByAltText('Venue Image Preview'),
+      ).not.toBeInTheDocument();
     });
 
-    test('removes image preview when clear button is clicked', async () => {
+    test('removes image preview when clear button is clicked and tests fileInputRef is null', async () => {
+      // Create component with custom fileInputRef mock
+      const originalUseRef = React.useRef;
+      let refValue = { current: document.createElement('input') };
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'refValue' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)